### PR TITLE
Added app and test layer and README.md to emmtypingtool 0.0.1

### PR DESCRIPTION
**Pull request by mndoucette**
- Added app and test layer
- Added tool-specific README.md

### CODE REVIEW

File: emmtypingtool/0.0.1/Dockerfile [hunk 1]
```diff
@@ -65,4 +63,29 @@ RUN mkdir /db && cd /db && \
...
+# Download fastq files for test
+RUN mkdir /test/raw_reads && cd /test/raw_reads && \
+fasterq-dump --split-files ${SRA_ACCESSION} && \
+gzip ${SRA_ACCESSION}_*
+
+WORKDIR /data
+
+# Run emmtypingtool on test files
+RUN mkdir /data/test && \
+python /emm-typing-tool/emm_typing.py -m /db -i /test/raw_reads -o /data/test
```
Comment (erinyoung) [inline emmtypingtool/0.0.1/Dockerfile:91]: I like this test:
Downloads fastq files for ERR10762520 and runs them through /emm-typing-tool/emm_typing.py

File: emmtypingtool/0.0.1/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- sratoolkit 3.0.2
+
+## Example Usage
+
+```bash
+# Running emmtypingtool on /raw_reads directory containing fastq files (.fastq.gz)
+python emm_typing.py -m /db -i /raw_reads -o /output_directory
+
+# Running emmtypingtool on specified fastq files (sample1)
+python emm_typing.py -m /db -1 sample1.R1.fastq.gz -2 sample1.R2.fastq.gz -o /output_directory
```
Comment (erinyoung) [inline emmtypingtool/0.0.1/README.md:26]: I do have some concerns about your readme versus the test. 

You are adding everything in `/emm-typing-tool` to PATH. Are all the *.py files there not executable?

I think your readme will probably be 
```
python /emm-typing-tool/emm_typing.py -m /db -i /raw_reads -o /output_directory
```
Or 

```
emm_typing.py -m /db -i /raw_reads -o /output_directory
```

**Comment by erinyoung** [comment]
It looks like you are missing a label : dockerfile.version

Could you add the following line?
```
LABEL dockerfile.version="2" 
```

I think it would be near line 6

File: emmtypingtool/0.0.1/Dockerfile [hunk 3]
```diff
@@ -46,8 +45,7 @@ RUN wget 'ftp://emboss.open-bio.org/pub/EMBOSS/EMBOSS-6.6.0.tar.gz' && \
   chmod -R 777 emboss
 
 # Install python packages
-RUN pip install --upgrade pip && \
-  python -m pip install pyyaml==5.1.2 numpy==1.16.4 lxml==4.4.1 biopython==1.74
+RUN python -m pip install pyyaml==5.1.2 numpy==1.16.4 lxml==4.4.1 biopython==1.74
 
 # Install PHE emm-typing-tool
 RUN git clone https://github.com/phe-bioinformatics/emm-typing-tool.git
```
[the remark this answers is not in the document]
Comment (mndoucette) [reply emmtypingtool/0.0.1/Dockerfile]: I added git checkout for the latest commit tag, 500d048ee49006d7aa3245fa4ee4ae7f99c398b4

File: emmtypingtool/0.0.1/Dockerfile (hunk 1 above)
Comment (mndoucette) [reply emmtypingtool/0.0.1/Dockerfile]: Thanks!

File: emmtypingtool/0.0.1/README.md (hunk 2 above)
Comment (mndoucette) [reply emmtypingtool/0.0.1/README.md]: I've updated the README.md with the correct example. Should be

`emm_typing.py -m /db -i /raw_reads -o /output_directory`

**Comment by mndoucette** [comment]
> It looks like you are missing a label : dockerfile.version
> 
> Could you add the following line?
> 
> ```
> LABEL dockerfile.version="2" 
> ```
> 
> I think it would be near line 6

**Review by erinyoung — APPROVED** [review]
Looks great!

I'm going to merge this and deploy to dockerhub and quay.

**Comment by erinyoung** [comment]
You can watch the progress here:

https://github.com/StaPH-B/docker-builds/actions/runs/4145858412

I'm going to warn you, though, that we're using this PR as a test for a new bot with quay. This deploy may fail, but likely not because of anything you've done.

**Pull request merged**